frontend/src/index.tsx

```diff
@@ -35,6 +36,16 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                   </ProtectedRoute>
                 ),
               },
+              {
+                path: AppRoute.MEDITATION_LIST,
+                element: (
+                  <ProtectedRoute>
+                    <SidebarWrapper>
+                      <MeditationPlayList />
```

Comment:
```suggestion
                      <MeditationList />
```

Reply:
I think file name also should be changed then

Reply:
yes, you can name the page as Meditation, but instead of MeditationTrack it's better to have MeditationEntry or MeditationRecord, because we can have videos too

Reply:
done